guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-manually.adoc

```diff
@@ -0,0 +1,63 @@
+[id="deploying-the-ca-certificate-on-a-host-manually"]
+= Deploying the CA certificate on a host manually
+
+You can deploy the CA certificate on the host manually by rendering a public provisioning template, which will provide the certificate to your host.
+
+.Prerequisites
+* You have root access on your host.
+* The CA certificate has been changed on {ProjectServer}.
+For more information, see {AdministeringDocURL}planning-for-ca-certificate-renewal[Planning for CA certificate renewal] in _{AdministeringDocTitle}_.
+
+.Procedure
+. Log in to you host by using SSH.
```

Comment:
Do we care how they access their host? Maybe this should drop the SSH or use `such as SSH`?